src/test_l1_dsDisplay.c

```diff
@@ -752,41 +724,44 @@ void test_l1_dsDisplay_negative_dsGetEDIDBytes(void) {
         return;
```

Comment:
remove if condition, use UT FATAL macros. For init and term use the FATAL macros. Do the same for other init/term calls
Buffer allocation of edid can be added after the init success.

Reply:
done